File: Layer_datas/Entities/activities/Place.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Layer_datas.Entities.activities
{
    [Table("activities_places")]
    public class Place : Entity
    {
        public string name { get; set; }
        public string adress { get; set; }
        public string postalcode { get; set; }
        public string city { get; set; }

        public virtual List<Session> sessions { get; set; }
    }
}

File: src/server/routes/graphql/schemas/Activities/place.ts
import { schema } from "./../schema"
import { UpdateArgs } from "routes/graphql/utils";
import { Models } from "datas-layer";
import { ICategoryAttributes } from "datas-layer/Entities/Activities/Category";

const place: schema<ICategoryAttributes> = {
    typeDefs: `
    type Place {
        id: Int
        name: String
        adress: String
        postalcode: String
        city: String
    }
    `
}

export default place;